# Metaphylogeny  

**Materials for the paper titled:**  
*Constructing Phylogenetic Trees for Microbiome Data Analysis: A Mini-Review*  
Published in *Computational and Structural Biotechnology Journal*  

## Overview  
- The **Supplementary** folder contains all supplementary materials and figures referenced in the paper.  
- Processed real data from 16S rRNA and WGS analyses (using QIIME 2, LotuS 2, MetaPhlAn 4, and Woltka) are stored in the **Phyloseq_objects** folder in **'phyloseq'** object format.  
- Code for visualizations included in the supplementary material is available in the **Code** folder as `Code_real_data.R`.  
- Code for visualizations in the main manuscript is located in the same **Code** folder as `Code_tree_visualization.R`.  

## Usage and Citation  
- If you use our processed data, **please cite our paper**.  
- Visualizations of taxonomy trees are based on the **BAZE** R package.  
  - Detailed tutorials for BAZE can be found at: [https://github.com/bioscinema/BAZE](https://github.com/bioscinema/BAZE)  
  - If you use the **BAZE** package, please cite both the package and our paper.

## Citation  
Please cite this article as: R. Liu, X. Qiao, Y. Shi et al., Constructing phylogenetic trees for microbiome data analysis: A mini-review, Computational and Structural Biotechnology
Journal, https://doi.org/10.1016/j.csbj.2024.10.032

## Contact  
- If you have any questions regarding the code, please contact Ruitao Liu at: **rxl761@case.edu**  
- If you are interested in collaborating with our lab or accessing additional microbiome data, please contact the corresponding author Liangliang Zhang at: **lxz@case.edu**
